app/src/main/java/chat/rocket/android/profile/ui/ProfileFragment.kt

```diff
@@ -51,6 +84,18 @@ class ProfileFragment : Fragment(), ProfileView, ActionMode.Callback {
     override fun showProfile(avatarUrl: String, name: String, username: String, email: String) {
         image_avatar.setImageURI(avatarUrl)
 
+        //click on image_avatar to change avatar
+        image_avatar.setOnClickListener(object : View.OnClickListener {
+            override fun onClick(view: View) {
+                val permissionCheck = ContextCompat.checkSelfPermission(context!!,
```

Comment:
context!!

Reply:
What changes need to be made here @filipedelimabrito ?